### ISSUE TITLE
Multivariate prediction univariate

### ISSUE THREAD

**Issue by a186232641** [OrigamiSL/FDNet2023#4]
hello, I would like to ask, how can this model be adjusted to apply to the multi-index prediction tasks involved?

**Comment by OrigamiSL** [comment]
You may change the weight of the last linear projection layer with a flatten operation before it, if necessary, to meet your need of output dimension. It is also feasible to add an additional linear projection or any other suitable module, if you want, to change the dimension of the variable to what you need.

**Comment by a186232641** [comment]
# parser.add_argument('--cols', type=str, nargs='+', help='certain cols from the data files as the input features') This parameter has no effect

**Comment by OrigamiSL** [comment]
True. That argument is useless in this work since it is not needed in experiments.
